aladdin/lib/version_check.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import os
+import sys
+
+from aladdin import __version__
+from aladdin.lib import in_aladdin_container, utils
+from aladdin.lib.git import Git
+from aladdin.config import load_config, ALADDIN_DEV, PROJECT_ROOT
+
+
+def check_latest_version():
+    if in_aladdin_container():
+        return
+    config = load_config()
+    repo = config["aladdin"]["repo"]
+    tag = config["aladdin"]["tag"]
+    enforce_version = (
+        config["aladdin"].get("enforce_version", False)
+        and not ALADDIN_DEV
+    )
+    git_url = f"git@github.com:{repo}.git"
+
+    _aladdin_version_check(tag, git_url, enforce_version=enforce_version)
+    _config_version_check(tag, enforce_version=enforce_version)
+    _plugins_version_check(tag, enforce_version=enforce_version)
+
+
+def _aladdin_version_check(tag, git_url, enforce_version=False):
+    live = Git.get_hash_ls_remote(tag, git_url, "--tags")
+    expected = Git.get_hash_ls_remote(__version__, git_url, "--tags")
```

Comment:
this one is slightly more lax than the other checks since there's no good way to know the current aladdin git hash, so we use the version number and check that git hash